.gitignore

```diff
@@ -1 +1,2 @@
 *~
+recipes-apps/xcsoar/.DS_Store
```

Comment:
This file (`.DS_Store`) is part of your operating system, it should not be part of the project. Ignore it in your user-wide ignore file in `~/.gitignore`.